falcon/media/json.py

```diff
@@ -69,9 +69,19 @@ def __init__(self, dumps=None, loads=None):
         self.dumps = dumps or partial(json.dumps, ensure_ascii=False)
         self.loads = loads or json.loads
 
+        # PERF(kgriffs): Test dumps once up front so we can just quickly
+        #   check the flag later.
```

Comment:
Nitpick: this note is slightly out of date now, since there is no longer any flag _per se_.